authors/apps/profiles/views.py

```diff
@@ -67,3 +70,125 @@ class AuthorsAPIView(ListView):
     queryset = User.objects.all()
     serializer_class = UserSerializer
     renderer_classes = (UserJSONRenderer,)
+
+
+class FollowsView(APIView):
+    permission_classes = (permissions.IsAuthenticated,)
+
+    def post(self, request, username):
+        """
+        User follows an Author
+        """
+        follower_id = User.objects.get(username=request.user.username).id
+        try:
+            followed_id = User.objects.get(username=username).id
+            self.profile_id = Profile.objects.get(user_id=followed_id).id
+            self.check_follow_status(
+                follower_id, followed_id, username)
+        except Exception as e:
```

Comment:
Thanks @joelethan for the good work,However catching a bare Exception is very expensive performance wise, so instead of checking the user doesnot exist in the exception block,you can just catch that.
You could have
 ``` 
except User.DoesNotExist as e:
```